ui/src/admin/components/DeprecationWarning.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {SFC} from 'react'
+
+interface Props {
+  message: string | JSX.Element
```

Comment:
are we currently using this where message would be a string?

Reply:
Nope. That is leftover from when i originally made the component. i will change to just JSX.Element